be/src/exec/line_reader.h

```diff
@@ -24,7 +24,7 @@ namespace doris {
 namespace io {
 class IOContext;
 }
-// This class is used for CSV scanner, to read content line by line
+// This class is used to read content line by line
 class LineReader {
```

Comment:
Split this PR into two PRS:
one pr for refactoring, the other PR for adding support enclose and escape for stream load.

Reply:
I'll try to.